pipelines/managed/push-rpm-to-koji/push-rpm-to-koji.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: push-rpm-to-koji
   labels:
-    app.kubernetes.io/version: "0.1.0"
+    app.kubernetes.io/version: "0.2.0"
```

Comment:
Maybe there are more changes coming, but if not, you don't need to change this version (or update the pipeline's README) since nothing in changing in this yaml - only in the task (which you've done)

Reply:
The new task is included in the pipeline, hence the version change - there are some backwards incompatible changes.

Reply:
There is no change to this yaml. Look at the diff - it is solely the version line. We don't update the pipeline versions if a task inside of them is updated - only if we add/remove a task from the pipeline or update parameters passed to one of the tasks

Reply:
> There is no change to this yaml. Look at the diff - it is solely the version line.

The YAML file did not change, but the pipeline uses a different task. I.e. there was a change in the pipeline itself. As a user of a pipeline (not a separate task) a would like to know when the pipeline changes (not some YAML file specifically) - especially when it contains a breaking changes.

Reply:
OK. Reverted.